shell/android-studio/flycast/build.gradle

```diff
@@ -18,6 +18,7 @@ def getSentryUrl = { ->
 
 android {
     namespace 'com.reicast.emulator'
+    ndkVersion '23.1.7779620'
```

Comment:
- NDK r23 is the latest NDK to support Jelly Bean. Source: https://github.com/android/ndk/wiki/Changelog-r23.
- AGP 8.0 uses NDK 25.1.8937393 by default (source: https://developer.android.com/build/releases/gradle-plugin#8-0-0) which is why I force the version here.